addons/website/static/src/components/dialog/dialog.xml

```diff
@@ -12,10 +12,10 @@
         <t t-if="props.showFooter" t-set-slot="footer">
             <t t-if="props.slots and props.slots.footer" t-slot="footer"/>
             <t t-else="">
-                <button class="btn btn-primary" t-on-click="protectedClick(props.primaryClick)" t-att-disabled="state.disabled">
+                <button class="btn btn-primary" t-on-click="protectedClick(props.primaryClick)" t-att-disabled="state.disabled" data-hotkey="q">
                     <t t-esc="props.primaryTitle"/>
                 </button>
-                <button t-if="props.showSecondaryButton" class="btn btn-secondary" t-on-click="protectedClick(props.secondaryClick)" t-att-disabled="state.disabled">
+                <button t-if="props.showSecondaryButton" class="btn btn-secondary" t-on-click="protectedClick(props.secondaryClick)" t-att-disabled="state.disabled" data-hotkey="x">
```

Comment:
Like I wrote on the [PR with framework changes](https://github.com/odoo/odoo/pull/183709):

> I chose "Q" and "X" because these are the hotkeys on action view dialogs.
> It could be "S" and "J" like some other dialogs in the frontend.
> It could be "C" and "J" like the FormViewDialog.
> 
> I don't have any opinion on that, except that it would be a good idea 1) to standardize all our dialogs instead of choosing different letters half the time, 2) to make sure hotkeys are used everywhere, else we can't count on them and they're useless. 🤷